<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>492</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>144</green>
        <blue>134</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>144</green>
        <blue>134</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>144</green>
        <blue>134</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>144</green>
        <blue>134</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Generar transaccion</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblTitulo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>GENERAR TRANSACCION</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLogo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../.designer/backup/logobanco.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lblContrasea">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cuenta origen:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCuentaOr">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblContrasea_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cuenta destino:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCuentaDes">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblContrasea_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Monto:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="SBMonto">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <double>999999999999.989990234375000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConfirmar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>CONFIRMAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>440</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLogoU">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logobanco.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lblContrasea_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>501</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Â¡Se pueden referenciar a las cuentas con su cbu o alias!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
